frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -457,23 +461,16 @@ export default class FirebaseInteractor {
     for (let word of intervention.wordList as string[]) {
       // Get the word
       let actualWord = await this.getWord(word);
-      let wordRef = this.db
-        .collection("words")
-        .doc(word)
-        .collection("intervention-set");
-      // Get each activity from firebase
-      let activity1 = (
-        await wordRef.doc("activity1").get()
-      ).data() as Definition;
-      let activity2 = (await wordRef.doc("activity2").get()).data() as Example;
-      let activity3 = (await wordRef.doc("activity3").get()).data() as Context;
-      let activity4 = (await wordRef.doc("activity4").get()).data() as Review;
-      let activity3Part2 = (
-        await wordRef.doc("activity3-part2").get()
-      ).data() as Context;
-      let activity3Part3 = (
-        await wordRef.doc("activity3-part3").get()
-      ).data() as Context;
+      let {
+        activity1,
+        activity2,
+        activity3,
+        activity3Part2,
+        activity3Part3,
+        activity4,
+      } = (await this.db.collection("words").doc(word).get()).data()
```

Comment:
yeah this looks a lot nicer than before -- nice cleanup here and everywhere else :o